internals/overlord/backend.go

```diff
@@ -34,3 +34,4 @@ func (osb *overlordStateBackend) Checkpoint(data []byte) error {
 func (osb *overlordStateBackend) EnsureBefore(d time.Duration) {
 	osb.ensureBefore(d)
 }
+
```

Comment:
Is this a deliberate addition of a newline?

Reply:
It's accidental, good catch!